public/images/socials/facebook-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="11" height="18" viewBox="0 0 11 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Will prefer you use react icons instead, it's already included as a dependency.

Reply:
Noted, thanks